README.md

```diff
@@ -69,6 +69,30 @@ Brief description of the named fields:
 * **Cluster name** - name of the cluster to create or to switch to if a raw kubeconfig was provided
 * **Certificate of certificate authority** - an optional certificate to check the Kubernetes api server's against
 
+### Using namespaces
+In order to use namespaced contexts you have to upload raw .kube/config to Jenkins credentials menu and then use it as a value to `credentialsId`.
+
+1. Create new credentials in Jenkins, choose i.e. 'Secret file' option.
```

Comment:
This is an explanation on how to upload credentials using the web interface, rather than describing how to access different namespaces of a raw kubeconfig file. Let's remove the sentence `In order ..`, rename the section `Uploading credentials with the web-interface` and move it before the `Pipeline usage` section